AutoRest/TestServer/server/routes/number.js

```diff
@@ -25,8 +25,21 @@ var number = function(coverage) {
             } else {
                 utils.send400(res, next, "Did not like the value provided for big double in the req " + util.inspect(req.body));
             }
```

Comment:
Hi @amarzavery,
Let me know if these changes look okay - then once the PR is merged I'll open an issue for other languages :)